src/Datadog.Trace.ClrProfiler.Managed/Integrations/WcfIntegration.cs

```diff
@@ -139,10 +139,10 @@ private static Scope CreateScope(RequestContext requestContext)
                 }
 
                 scope = tracer.StartActive("wcf.request", propagatedContext);
-                Span span = scope.Span;
+                var span = scope.Span;
 
                 span.DecorateWebServerSpan(
-                    resourceName: requestMessage.Headers.Action,
+                    resourceName: requestMessage.Headers.Action ?? requestMessage.Headers.To.LocalPath,
```

Comment:
For the new fallback, it might be safer to do add the null conditional operator: `requestMessage.Headers.To?.LocalPath`. I don't know too much about WCF so I don't know if it's entirely necessary but it's a small safeguard and we do this below for the HTTP URI below.

Reply:
I see, so if `LocalPath` is null it won't try to access it, right? Seems safe, thanks!